db/migrate/20211130133210_add_index_for_nudge_job.rb

```diff
@@ -0,0 +1,13 @@
+class AddIndexForNudgeJob < ActiveRecord::Migration[6.1]
+  def change
+    add_index :solutions, [:last_iterated_at, :user_id], name: "fast_1"
+    add_index :solutions, [:user_id, :last_iterated_at], name: "fast_2"
+    add_index :solutions, [:type, :user_id, :exercise_id, :last_iterated_at], name: "fast_3"
+    add_index :solutions, [:exercise_id, :type, :user_id, :last_iterated_at], name: "fast_4"
```

Comment:
Will this temporarily lock the table? I'm not sure how mysql does this.

Reply:
Not if you use the algorithm+lock commands as per https://github.com/exercism/website/pull/2197/files#diff-e85712804207d693a50b13df15695b213427e6f51c2eab47a534762de03a3230R3 - Mysql now supports non-blocking table modifications if those are specified.

Reply:
Cool